.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "Go",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": {
+			"INSTALL_AVALANCHEGO_VERSION": "d729e5c7ef9f008c3e89cd7131148ad3acda2e34",
+			"INSTALL_AVALANCHEGO_VERSION_SHORT": "d729e5c7"
```

Comment:
Could we switch from using a commit to a specific tag of AvalancheGo, so that it's more readable? We'll then need to keep this up to date when updating AvalancheGo as well